tests/fixtures/bug_features/comment_length.json

```diff
@@ -1,2 +1,2 @@
s.mozilla.com/orangefactor/?display=Bug&bugid=1383552&startday=2017-12-11&endday=2017-12-17&tree=all", "id": 12923114, "creation_time": "2017-12-18T01:27:59Z", "author": "orangefactor@bots.tld"}, {"text": "https://wiki.mozilla.org/Bug_Triage#Intermittent_Test_Failure_Cleanup", "id": 12973895, "creation_time": "2018-01-16T22:42:29Z", "author": "bug-husbandry-bot@mozilla.bugs"}, {"text": "https://treeherder.mozilla.org/logviewer.html#?job_id=208166778&repo=mozilla-central&lineNumber=73448", "id": 13716380, "creation_time": "2018-10-27T11:32:31Z", "author": "ncsoregi@mozilla.com"}, {"text": "1 failures in 2584 pushes (0.0 failures/push) were associated with this bug in the last 7 days.\n\nRepository breakdown:\n* mozilla-central: 1\n\nPlatform breakdown:\n* windows7-32: 1\n\nFor more details, see:\nhttps://treeherder.mozilla.org/intermittent-failures.html#/bugdetails?bug=1383552&startday=2018-10-22&endday=2018-10-28&tree=all", "id": 13717871, "creation_time": "2018-10-29T07:11:01Z", "author": "orangefactor@bots.tld"}, {"text": "1 failures in 2917 pushes (0.0 failures/push) were associated with this bug in the last 7 days.\n\nRepository breakdown:\n* autoland: 1\n\nPlatform breakdown:\n* windows7-32: 1\n\nFor more details, see:\nhttps://treeherder.mozilla.org/intermittent-failures.html#/bugdetails?bug=1383552&startday=2018-11-12&endday=2018-11-18&tree=all", "id": 13764050, "creation_time": "2018-11-19T07:03:10Z", "author": "orangefactor@bots.tld"}, {"text": "1 failures in 2917 pushes (0.0 failures/push) were associated with this bug in the last 7 days.\n\nRepository breakdown:\n* autoland: 1\n\nPlatform breakdown:\n* windows7-32: 1\n\nFor more details, see:\nhttps://treeherder.mozilla.org/intermittent-failures.html#/bugdetails?bug=1383552&startday=2018-11-12&endday=2018-11-18&tree=all", "id": 13765378, "creation_time": "2018-11-19T17:21:04Z", "author": "orangefactor@bots.tld"}, {"text": "1 failures in 1862 pushes (0.001 failures/push) were associated with this bug in the last 7 days.\n\nRepository breakdown:\n* mozilla-inbound: 1\n\nPlatform breakdown:\n* windows7-32: 1\n\nFor more details, see:\nhttps://treeherder.mozilla.org/intermittent-failures.html#/bugdetails?bug=1383552&startday=2018-12-03&endday=2018-12-09&tree=all", "id": 13802912, "creation_time": "2018-12-10T06:19:30Z", "author": "orangefactor@bots.tld"}, {"text": "2 failures in 2648 pushes (0.001 failures/push) were associated with this bug in the last 7 days.\n\nRepository breakdown:\n* mozilla-central: 1\n* autoland: 1\n\nPlatform breakdown:\n* windows7-32: 2\n\nFor more details, see:\nhttps://treeherder.mozilla.org/intermittent-failures.html#/bugdetails?bug=1383552&startday=2018-12-17&endday=2018-12-23&tree=all", "id": 13829666, "creation_time": "2018-12-24T06:15:21Z", "author": "orangefactor@bots.tld"}, {"text": "https://wiki.mozilla.org/Bug_Triage#Intermittent_Test_Failure_Cleanup", "id": 13871675, "creation_time": "2019-01-18T22:24:54Z", "author": "bug-husbandry-bot@mozilla.bugs"}]}
```

Comment:
As this is a test fixture, you can keep the old link here and avoid changing this file and tests/test_bug_features.py